src/board.cpp

```diff
@@ -0,0 +1,166 @@
+#include <iostream>
+#include <vector>
+#include <string>
+#include "board.h"
+#include "gui.h"
+
+Board::Board() : Board(9, false) {}
+
+Board::Board(size_t size, bool cpu) {
+  isCpu = cpu;
+  p1Score = 0;
+  p2Score = 0;
+  boardSize = size;
+  for (int i = 1; i <= size; i++) {
+    p1.insert(i);
+    p2.insert(i);
```

Comment:
not sure but it might be better to have a list of players rather than having to do p1 and p2

Reply:
Having a list of players might be a good idea. I might make a multiplayer version of this in the future but for now, just PVP. Good idea to scale it though. A player class regardless would be useful for this.

Reply:
This was utter pain to implement.